packages/e2e-tests/specs/experiments/blocks/navigation.test.js

```diff
@@ -784,4 +784,92 @@ describe.skip( 'Navigation', () => {
 
 		expect( isScriptLoaded ).toBe( true );
 	} );
+
+	describe( 'Creating and restarting', () => {
```

Comment:
Navigation e2e tests are currently all skipped, so this won't run.

I did start rewriting them completely, but have been too busy to finish up (plus everything is changed too rapidly to keep up.)

Reply:
Ah yes we forgot that. I pulled in from Adam's PR. We can leave it in for now especially since you unfocused it.